kit/brain_board/updating-the-brain.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: page
+title: Updating The Brain
+---
+
+{% assign latest_version = site.data.kit_versions | first %}
+{% assign latest_version = latest_version.version | default: "2022.1.0" %}
+
+# Updating The Brain
+
+### Flashing SD card
```

Comment:
Does the heading structure still make sense now its on its own page?

I would have thought Flashing SD Card would be h2, Etcher h3 and Flashing h4.